errors/runconfig.go

```diff
@@ -0,0 +1,204 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/daemon component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeInvalidMacAddress is generated when invalid Mac Address is provided.
+	ErrorCodeInvalidMacAddress = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDMACADDRESS",
```

Comment:
For example, `INVALIDMEMORYSWAPPINESS` should be `RUNCONFIG_INVALID_MEMORY_SWAPPINESS`. Either way, these error codes are way to detailed for use over an API. They don't provide any method to group the errors by how to handle them.